_maps/configs/pirate_apostolis

```diff
@@ -0,0 +1,47 @@
+{
+	"map_name": "Starwalker-class carrier",
```

Comment:
"Starwalker" is actually already a standard ship name in the general namelist. This may cause some confusion.

Reply:
I dont think this is a huge issue tbh

Reply:
It's also a bit jokey and doesn't really sound appropriate/piratey, tbh, but those are personal gripes.